lua-tree/share/lua/5.1/kong/db/schema/typedefs.lua

```diff
@@ -373,13 +364,13 @@ typedefs.sni = Schema.define {
 
 typedefs.certificate = Schema.define {
   type = "string",
-  custom_validator = validate_certificate,
```

Comment:
This requires updating the [patch](https://github.com/Kong/goks/tree/main/patches) file.

Reply:
yes, one of the open checkpoints.   is there any script for that?  like a reverse `make gen-lua-tree`

Reply:
Unfortunately no :-( What I do is manually edit the patch making sure the diff notation is correct, add the Lua files that are being updated, and then validate with the `make gen-lua-tree` to make sure no other changes come in and the patch is successful.

We should probably invest some time to make this easier and for some of the sweeping changes that are coming in 3.0, but for now there isn't anything automated.

Reply:
done